test/integration/offchain/match2.test.ts

```diff
@@ -217,6 +220,31 @@ describe('match2', () => {
         transactionType: 'cancellation',
       })
     })
+
+    it('gets all cancellation transactions for a specific match2', async () => {
+      const { status, body } = await get(app, `/v1/match2/${seededMatch2Id}/cancellation`)
+
+      expect(status).to.equal(200)
+      expect(body).to.be.an('array')
+      expect(body.find(({ id }: { id: string }) => id === seededMatch2CancellationId)).to.deep.equal({
+        id: seededMatch2CancellationId,
+        apiType: 'match2',
+        transactionType: 'cancellation',
+        localId: seededMatch2Id,
+        state: 'submitted',
+        submittedAt: exampleDate,
```

Comment:
for example date we can use `mockdate` lib https://www.npmjs.com/package/mockdate so no need to do hardcore prefixes/hardcoding

Reply:
Can I do this? I think the hardcoded date is important to know when we are doing the tests and we are filtering by records made before or after a certain date - or should I change the tests to use mockdate too?